gendiff/scripts/gendiff.py

```diff
@@ -16,7 +45,11 @@ def main():
         help="set format of output",
     )
     args = parser.parse_args()
-    print(parser, args)
+
```

Comment:
as we are doing three things now (reading args, creating diff and printing it), consider refactoring `main` to call a function that returns args from argparse, then call function to generate diff and then do print of the result  

smth like this  
```python
def main() -> None:
    args = ...()
    diff = ...()
    print(diff)
```